src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Presentation/ImageMonikers.cs

```diff
@@ -0,0 +1,284 @@
+                        switch (GetAccessibility(tags))
+                        {
+                            case Accessibility.Public:
+                                return KnownMonikers.ModulePublic;
+                            case Accessibility.Protected:
+                                return KnownMonikers.ModulePublic;
+                            case Accessibility.Private:
+                                return KnownMonikers.ModulePrivate;
+                            case Accessibility.Internal:
+                                return KnownMonikers.ModuleInternal;
+                            default:
+                                return KnownMonikers.Module;
+                        }
+
+                    case CompletionTags.Folder:
+                        return KnownMonikers.OpenFolder;
+
+                    case CompletionTags.Operator:
+                        return KnownMonikers.Operator;
+
+                    case CompletionTags.Parameter:
+                        return KnownMonikers.FieldPublic;
+
+                    case CompletionTags.Property:
+                        switch (GetAccessibility(tags))
+                        {
+                            case Accessibility.Public:
+                                return KnownMonikers.PropertyPublic;
+                            case Accessibility.Protected:
+                                return KnownMonikers.PropertyProtected;
+                            case Accessibility.Private:
+                                return KnownMonikers.PropertyPrivate;
+                            case Accessibility.Internal:
+                                return KnownMonikers.PropertyInternal;
+                            default:
+                                return KnownMonikers.Property;
+                        }
+
+                    case CompletionTags.RangeVariable:
+                        return KnownMonikers.FieldPublic;
+
+                    case CompletionTags.Reference:
+                        return KnownMonikers.Reference;
+
+                    case CompletionTags.Structure:
+                        switch (GetAccessibility(tags))
+                        {
+                            case Accessibility.Public:
+                                return KnownMonikers.StructurePublic;
+                            case Accessibility.Protected:
+                                return KnownMonikers.StructureProtected;
+                            case Accessibility.Private:
+                                return KnownMonikers.StructurePrivate;
+                            case Accessibility.Internal:
+                                return KnownMonikers.StructureInternal;
+                            default:
+                                return KnownMonikers.Structure;
+                        }
+
+                    case CompletionTags.TypeParameter:
+                        return KnownMonikers.Type;
```

Comment:
This seems wonky.  I could have sworn there was some "<T>" glyph that was in VS for this purpose.  I believe C++ uses it for their template parameters as well.